packages/ecc-client-ga4gh-tes/src/components/tes-get-run/definition/tesGetRun.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  FASTElement,
+  attr,
+  customElement,
+  observable,
+} from '@microsoft/fast-element';
+import template from './tesGetRun.template.js';
+import styles from './tesGetRun.styles.js';
+import { fetchTask } from '../../../data/index.js';
+import { deleteTask } from '../../../data/Post/tesPost.js';
+import TaskData from './TaskData.js';
+
+@customElement({
+  name: 'ecc-tes-get-run',
+  template,
+  styles,
+  shadowOptions: { mode: 'open' },
+})
+export default class TESGetRun extends FASTElement {
+  @attr id = '';
+
+  @attr state = '';
+
+  @observable isLoading = true;
+
+  @observable data: TaskData = {
+    id: this.id,
+    state: this.state,
+    name: '',
+    description: '',
+    creation_time: '',
+    executors: [],
+    logs: [],
+  };
+
+  connectedCallback() {
+    super.connectedCallback();
+    // Add event listener to handle accordion open event
+    this.addEventListener('change', this.handleFetch);
+    const delButton = this.shadowRoot?.querySelector('fast-button');
```

Comment:
I'm pretty sure this will always return null, cause the parent component mounts before it's children.

Reply:
done, please check and resolve